src/config.c

```diff
@@ -797,37 +797,47 @@ void configSetCommand(client *c) {
  *----------------------------------------------------------------------------*/
 
 void configGetCommand(client *c) {
-    robj *o = c->argv[2];
     void *replylen = addReplyDeferredLen(c);
-    char *pattern = o->ptr;
     int matches = 0;
-    serverAssertWithInfo(c,o,sdsEncodedObject(o));
+    int i;
 
     for (standardConfig *config = configs; config->name != NULL; config++) {
-        /* Hidden configs require an exact match (not a pattern) */
-        if (config->flags & HIDDEN_CONFIG) {
-            if (!strcasecmp(pattern, config->name)) {
+        int matched_conf = 0;
+        int matched_alias = 0;
+        for (i = 0; i < c->argc - 2 && (!matched_conf || !matched_alias); i++) {
+            robj *o = c->argv[2+i];
+            char *pattern = o->ptr;
+
+            /* Hidden configs require an exact match (not a pattern) */
+            if (config->flags & HIDDEN_CONFIG) {
+                if (!matched_conf && !strcasecmp(pattern, config->name)) {
```

Comment:
i wanna suggest a refactoring to share the code of hidden configs and non-hidden configs.
i.e. use a different string compare function, but all the code inside that if (in case it matches), can be shared.
WDTY?

Reply:
Done. Let me know if it looks any better.